### PR TITLE
docs: Updates following runthrough pt1 

### PR DESCRIPTION
**Pull request by Callisto13**
Part of https://github.com/weaveworks/weave-gitops-enterprise/issues/2274

3 commits:

docs: Update getting started screenshots

------------------------------

docs: Static/cluster/test user -> emergency user

This user is referred to by 3 different names. I have now named it as
the 'emergency user', since it is something we do not recommend outside
of test environments. I have given it it's own doc and split it from the
OIDC page.

This commit also contains other bits of "tidying-up" for the
'configuration/' folder.

------------------------------

docs: Minor changes to docs/guides

### CODE REVIEW

File: website/docs/configuration/emergency-user.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    resources: ["*"]
+    verbs: [ "get", "list", "watch" ]
+```
+
+These permissions give the emergency user Administrator level powers. **We do not
+advise leaving it active on production systems**.
+
+:::tip
+To disable the emergency user, set `adminUser.create` to `false` when deploying
+the [Helm Chart](../references/helm-reference.md).
```
Comment (makkes) [inline website/docs/configuration/emergency-user.mdx:87]: Unfortunately this is not enough. As I described in [this comment](https://github.com/weaveworks/weave-gitops/issues/3263#issuecomment-1420604127) you need to explicitly set the `--auth-methods` flag to completely disable logging in through the emergency user Secret.

If the Secret has been created once, setting `adminUser.create` will not delete it so in order to actually disable the emergency user you'd have to either delete the Secret or set the `--auth-methods` flag accordingly.

Comment (Callisto13) [reply website/docs/configuration/emergency-user.mdx:87]: Ahhh good to know ty! Could not find that in the current docs ⭐

File: website/docs/configuration/emergency-user.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+Follow the instructions in the next section in order to configure RBAC correctly.
+
+:::tip
+To remove the emergency user as a login method, set the following values in the
+[Helm Chart](../references/helm-reference.md):
+
+```yaml
+#
+adminUser:
+	create: false
```
Comment (makkes) [inline website/docs/configuration/emergency-user.mdx:96]:
```suggestion
    create: false
```

**Review by makkes — COMMENTED** [review]
Perfect, just a tiny formatting suggestion.

File: website/docs/configuration/emergency-user.mdx (hunk 2 above)
Comment (Callisto13) [reply website/docs/configuration/emergency-user.mdx]: 👍 fixed

**Review by yitsushi — APPROVED** [review]

File: website/docs/intro.mdx [hunk 3]
```diff
@@ -40,7 +40,7 @@ OSS Weave GitOps has a number of awesome features to take your team beyond a sim
 - :mag: Uncover relationships between resources and quickly navigate between them.
 - :thinking_face: Understand how workloads are reconciled through a directional graph.
 - :goggles: View Kubernetes events relating to a given object to understand issues and changes.
-- :no_pedestrians: Secure access to the dashboard through the ability to integrate with an OIDC provider (such as Dex) or through a configurable cluster user.
+- :no_pedestrians: Secure access to the dashboard through the ability to integrate with an OIDC provider (such as Dex).
```
Comment (serboctor) [inline website/docs/intro.mdx:43]: should we mention the emergency user mentioned above?

**Review by serboctor — COMMENTED** [review]
LGTM

File: website/docs/intro.mdx (hunk 3 above)
Comment (Callisto13) [reply website/docs/intro.mdx:43]: I'd rather that nobody used it ever, so it's not a good feature to highlight in the intro imo

**Pull request merged**